### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by NikolaGolubovic**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "nodejs-1",
+  "version": "1.0.0",
+  "description": "Hello world",
+  "main": "index.js",
+  "scripts": {
+    "build": "tsc --outDir build",
```
Comment (vuxor) [inline package.json:7]: You should add build folder to gitignore

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import app from "./app";
+
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const PORT = process.env.PORT || 5000;
+
+app.listen(PORT, () => {
```
Comment (vuxor) [inline src/index.ts]: Connect main app with sequelize

**Review by vuxor — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 2 above)
Comment (NikolaGolubovic) [reply src/index.ts]: I will need some time to figure out sequilize and sequilize through typescript :)

Comment (NikolaGolubovic) [reply src/index.ts]: Made my solution in large with analyzing Vladimir's code,  
hope that in the future I will be much more original :)

Comment (vuxor) [reply src/index.ts]: Sure. Take some time and check documentation.

**Pull request merged**